state-chain/pallets/cf-environment/src/lib.rs

```diff
@@ -205,6 +207,16 @@ pub mod pallet {
 	#[pallet::getter(fn sol_vault_address)]
 	pub type SolanaVaultAddress<T> = StorageValue<_, SolAddress, ValueQuery>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn sol_genesis_hash)]
+	pub type SolanaGenesisHash<T> = StorageValue<_, SolHash, OptionQuery>;
+
+	#[pallet::storage]
+	#[pallet::getter(fn supported_sol_assets)]
```

Comment:
for other chains (eth, arb) they are actually called `supported_*_assets` so actually for consistency I did it that way, otherwise Solana will be the only one named like that.